lsp-ido.el

```diff
@@ -0,0 +1,134 @@
+;;; lsp-ido.el --- `ido' integration -*- lexical-binding: t -*-
```

Comment:
I think you should add this file here as well: https://github.com/emacs-lsp/lsp-mode/blob/master/Cask#L15

Reply:
Added.